.ci/bindist/linux/snap/Dockerfile

```diff
@@ -0,0 +1,67 @@
+# This Dockerfile is:
+#
+#   https://github.com/snapcore/snapcraft/pull/3384
+#   + xenial => focal
+#
+FROM ubuntu:focal as builder
+
+# Grab dependencies
+RUN apt-get update
+RUN apt-get dist-upgrade --yes
```

Comment:
Again, do we need this `dist-upgrade`?

Reply:
Probably not, it's copied 1:1 from Canonical's `Dockerfile`